### PR TITLE
TDAP055-Backend-Connect-Delete-Todos-And-Tost-Messages

### PR DESCRIPTION
**Pull request by amit-ib**
1. Added delete functionality using API
2. Prepared component for tost message
3. Added tost message on delete and edit

### CODE REVIEW

File: src/assets/scss/_common.scss [hunk 1]
```diff
@@ -28,3 +28,21 @@ label {
   display: block;
   font-size: 14px;
 }
+.alert {
```
Comment (akash-dhande-ib) [inline src/assets/scss/_common.scss:31]: This can be moved to respective file.

File: src/components/shared/Tost.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+interface Props {
+  delay: number;
+  //children: React.ReactNode;
+  setTost: React.Dispatch<React.SetStateAction<tostType>>;
+  tostMessage: string;
+  tostType: string;
+}
+const Tost = ({ delay, setTost, tostMessage, tostType }: Props) => {
+  useEffect(() => {
+    console.log("first");
```
Comment (akash-dhande-ib) [inline src/components/shared/Tost.tsx:13]: Debug

File: src/components/todo/TodoItem.tsx [hunk 3]
```diff
@@ -49,9 +51,20 @@ const TodoItem = ({ todoItem, setLoading }: Props) => {
...
 
   const [todo, setTodo] = useState(todoData);
-  const deleteTaskHandeler = () => {
-    dispatch(deleteTodoAction(todoItem));
+  const deleteTaskHandeler = async () => {
     setShowModal(false);
+    setLoading(true);
+    await axiosInstance.delete(`/todo/${todoItem.id}`).then(async () => {
+      await axiosInstance
+        .get("/todos")
```
Comment (akash-dhande-ib) [inline src/components/todo/TodoItem.tsx:59]: This looks bit mixed up, we can separate it for better readability.

**Review by akash-dhande-ib — APPROVED** [review]
Small recommendation, rest looks good.
Can be optimized later.

File: src/components/shared/Tost.tsx (hunk 2 above)
Comment (amit-ib) [reply src/components/shared/Tost.tsx]: Updated

File: src/assets/scss/_common.scss (hunk 1 above)
Comment (amit-ib) [reply src/assets/scss/_common.scss]: Moved to a separate file

File: src/components/todo/TodoItem.tsx (hunk 3 above)
Comment (amit-ib) [reply src/components/todo/TodoItem.tsx]: Updated

**Review by akash-dhande-ib — APPROVED** [review]

**Pull request merged**